### ISSUE TITLE
Strange errors in RPT

### ISSUE THREAD

**Issue by Tappczan** [A3Antistasi/antistasiofficial#255]
Hello, after my last save game went haywire, I started a new game with my friends on a dedicated server and this error keep poping up in the RPT log:

> 23:21:49 File Functions\fn_QRF_dismountTroops.sqf [AS_fnc_QRF_dismountTroops], line 45
> 23:21:50 Error in expression <; ((units _infGroup1 select 0) distance _dest < 50) || ({alive _x} count units _>
> 23:21:50   Error position: <_dest < 50) || ({alive _x} count units _>
> 23:21:50   Error Undefined variable in expression: _dest

This error spam started from this error:

> 19:36:36 Error in expression <andpos1 = [_dest, _posRoad, _dist] call findSafeRoadToUnload;private _vehData>
> 19:36:36   Error position: <findSafeRoadToUnload;private _vehData>
> 19:36:36   Error Undefined variable in expression: findsaferoadtounload
> 19:36:36 File mpmissions\__cur_mp.Altis\CREATE\enemyQRF.sqf, line 256
> 19:36:36 soldier[rhs_vdv_driver]:Some of magazines weren't stored in soldier Vest or Uniform?
> 19:36:37 soldier[rhs_vdv_arifleman]:Some of magazines weren't stored in soldier Vest or Uniform?
> 19:36:37 soldier[rhs_vdv_arifleman]:Some of magazines weren't stored in soldier Vest or Uniform?
> 19:36:37 "Transport dispatched to [3622.83,13779.5,0]"
> 19:36:37 Error in expression <tched to %1", _dest];
> [_grpVeh2, _orig, _landpos1, _mrk, [_grpDis1, _grpDis2], _>
> 19:36:37   Error position: <_landpos1, _mrk, [_grpDis1, _grpDis2], _>
> 19:36:37   Error Undefined variable in expression: _landpos1
> 19:36:37 File mpmissions\__cur_mp.Altis\CREATE\enemyQRF.sqf, line 290
> 19:36:37 Error in expression <then {
> _wp501 = _infGroup2 addWaypoint [_dest, 0];
> _wp501 setWaypointType "GETOU>
> 19:36:37   Error position: <_dest, 0];
> _wp501 setWaypointType "GETOU>
> 19:36:37   Error Undefined variable in expression: _dest
> 19:36:37 File Functions\fn_QRF_dismountTroops.sqf [AS_fnc_QRF_dismountTroops], line 27
> 19:36:37 Error in expression < 1;
> };
> 
> _wp400 = _vehGroup addWaypoint [_dest, 0];
> _wp400 setWaypointBehaviour ">
> 19:36:37   Error position: <_dest, 0];
> _wp400 setWaypointBehaviour ">
> 19:36:37   Error Undefined variable in expression: _dest
> 19:36:37 File Functions\fn_QRF_dismountTroops.sqf [AS_fnc_QRF_dismountTroops], line 18
> 19:36:37 Error in expression <s", "_duration"];
> 
> [_vehGroup, _origin, _dest, _mrk, _infGroups, _duration, "gro>
> 19:36:37   Error position: <_dest, _mrk, _infGroups, _duration, "gro>
> 19:36:37   Error Undefined variable in expression: _dest

After this, the QRF force sits in the base and don't attack anything. Going to check today if a complete server restart will help.

**Comment by Sparker95** [comment]
Probably it is related to this: https://github.com/A3Antistasi/antistasiofficial/issues/253

**Comment by StefArma** [comment]
is this last master version that are you playing?

**Comment by StefArma** [comment]
Oh actually this is propaganda mission, yes i forgot to repair it

**Comment by StefArma** [comment]
OK so, Chris coded this alternative qrf script called 
\Create\enemyqrf.sqf

i honestly have to read it through and understand what's different and why was it necessary for certain kind of istances like propagnda, devin, camp defend

**Comment by Tappczan** [comment]
Yep, that was on the latest 1.7.19B4. After the server restart the errors didn't show anymore in large numbers, but still sometimes after AAF counterattack is still shows.

**Comment by StefArma** [comment]
Thanks!

**Comment by StefArma** [comment]
Let me specify: theese QRF scripts were done in addition to the standard one. 
At some point we need to harmonize and make one suitable for different situations.
